incubator/skopeo-sync/values.yaml

```diff
@@ -0,0 +1,48 @@
+## -- A secret containing all of the authfiles necessary
+secretName: registry-creds
+
+## -- The number of successful finished jobs to retain. Value must be non-negative integer
+successfulJobsHistoryLimit: 0
+
+## -- The number of failed finished jobs to retain. Value must be non-negative integer.
+failedJobsHistoryLimit: 1
+
+## -- A list of syncs to use
+syncs:
+  - name: fargle
+    schedule: "2 5 * * *"
+    srcCred: gcr
+    src: us-docker.pkg.dev/registry/fargle
+    dst: quay.io/company/fargle
+  - name: bargle
+    schedule: "2 6 * * *"
+    dstCred: gcr
+    src: quay.io/company/bargle
+    dst: us-docker.pkg.dev/registry/bargle
```

Comment:
are these meant to be in the default `values.yaml`? Or in the ci test values?

Reply:
I wanted to leave them in defaults to show the structure of them.